src/diffusers/pipelines/hunyuandit/pipeline_hunyuandit.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2024 HunyuanDiT Authors and The HuggingFace Team. All rights reserved.
+# Copyright 2024 HunyuanDiT Authors, Qixun Wang and The HuggingFace Team. All rights reserved.
```

Comment:
can we make a `controlnet_hunyuan` folder and add a `pipeline_hunyuandit_controlnet.py`? (similarly to what we did with sd3 controlnet pipeline)

Reply:
Received, we will make the modifications according to the rules and logic of the sd3 controlnet. There have been quite a few sd3-related matters recently, and it may not be completed within the next few days. I apologize for the inconvenience.